index.bs

```diff
@@ -425,7 +432,61 @@ the API visibly relates to garbage collection details. By contrast, special APIs
 purpose such as `WeakRef` and `FinalizationGroup` have their GC interaction as a specific part of
 their contract with the developer.
 
-<h2 id="api-surface">JavaScript API Surface Concerns</h2>
+<h2 id="api-surface">API Surface Concerns</h2>
+
+<h3 id="naming-is-hard">Naming things</h3>
```

Comment:
please don't duplicate this whole section

Reply:
Addressed with the rebase.